File: demo/lang/lua/2_webpage_extract.lua
#!/usr/bin/env lua
--[[
title=extract content from webpage
cid=0
pid=0

1. Load web page from url http://xxx
2. Retrieve img element zt-logo.png in html
3. Check img exist >> `必应`

]]

require("socket")
local https = require("ssl.https")  -- need luasocket, luasec modules
local body, code, headers, status = https.request("https://cn.bing.com")

_, _, src = string.find(body, "<title>(.-)</title>")
print(src)

File: cmd/test/demo/php/2_webpage_extract.php
#!/usr/bin/env php
<?php

/**

title=extract content from webpage
timeout=0
cid=2
pid=1

1. Load web page from url http://xxx
2. Retrieve img element zt-logo.png in html
3. Check img exist >> `.*zt-logo.png`
1. Load web page from url http://xxx
2. Retrieve img element zt-logo.png in html
3. Check img exist >> `.*zt-logo.png`
1. Load web page from url http://xxx
2. Retrieve img element zt-logo.png in html
3. Check img exist >> `.*zt-logo.png`

*/

$resp = file_get_contents('http://max.demo.zentao.net/user-login-Lw==.html');
preg_match_all("/<img src=\"(.*)\" .*>/U", $resp, $matches);
echo $matches[1][0] . "\n";
print("abc123\n");